### DISCUSSION TITLE
ERROR while adding a new column in existing data frame

### DISCUSSION THREAD

**Discussion by bakhtawar-ishaq** [AammarTufail/python_ka_chilla2023#69] [General]
I used this code to add new column to dataset:

df["age_groups"]= pd.cut(df["age"],[0,10,18,30,50,80],labels=["Kids,Child,Youth,Middle aged,Old"])

But it is not working and showing error which says:
"ValueError: Bin labels must be one fewer than the number of bin edges"

can anyone tell why?

**Comment by usamabajwa86** [comment]
ADD inverted comas on each category

**Reply by usamabajwa86** [reply]
df["age_groups"]= pd.cut(df["age"],[0,10,18,30,50,80],labels=['Kids','Child','Youth','Middle_aged','Old'])

**Comment by bakhtawar-ishaq** [comment]
Thankyou so much Usma.
